frontend/app/src/components/home/proposal/MakeProposalModal.svelte

```diff
@@ -150,7 +150,7 @@
         busy = true;
 
         if (
-            !(await approvePayment(PROPOSALS_BOT_CANISTER, proposalCost + BigInt(2) * transferFee))
+            !(await approvePayment(PROPOSALS_BOT_CANISTER, proposalCost + transferFee))
```

Comment:
We only need to approve the proposal cost + a single fee, but then a fee gets charged on top of that for the approval and another fee gets charged again for the transfer.
So 3 fees are needed in total.